Plugins/BridgeJS/Sources/BridgeJSLink/BridgeJSLink.swift

```diff
@@ -644,11 +688,231 @@ struct BridgeJSLink {
                     printer.write("return value;")
                 }
                 printer.write("}")
+                printer.write("bjs[\"swift_js_get_optional_heap_object_pointer\"] = function() {")
+                printer.indent {
+                    printer.write("const pointer = \(JSGlueVariableScope.reservedStorageToReturnOptionalHeapObject);")
+                    printer.write("\(JSGlueVariableScope.reservedStorageToReturnOptionalHeapObject) = undefined;")
+                    printer.write("return pointer || 0;")
+                }
+                printer.write("}")
+
+                var closureSignatures: Set<ClosureSignature> = []
+                for skeleton in exportedSkeletons {
+                    collectClosureSignatures(from: skeleton, into: &closureSignatures)
+                }
+
+                var classToModule: [String: String] = [:]
+                for skeleton in exportedSkeletons {
+                    for klass in skeleton.classes {
+                        classToModule[klass.name] = skeleton.moduleName
+                    }
+                }
+
+                for signature in closureSignatures.sorted(by: { $0.mangleName < $1.mangleName }) {
+                    let invokeFuncName = "invoke_js_callback_\(signature.mangleName.lowercased())"
```

Comment:
To be conservative, it might be better not to force lowercase here to avoid conflicts with names having different casing